db/schema.rb

```
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[8.0].define(version: 2026_03_13_064334) do
+ActiveRecord::Schema[8.0].define(version: 2026_04_23_202332) do
```

RuboCop (1.86.1)

[convention] Use underscores(_) as thousands separator and separate every 3 digits with them.

(Style/NumericLiterals)